serverless/src/getRetrievals/handler.js

```diff
@@ -61,12 +61,15 @@ export default async function getRetrievals(event, context) {
         environment
       } = firstRow
 
+      const collectionMetadata = retrievalRecord.map((record) => record.collection_metadata)
+      const titles = collectionMetadata.map((collection) => ({ titles: collection }))
```

Comment:
`titles` isn't the right key for this, it needs to match the collection metadata key of `title`. The output of the tests should not have changed.

You are still returning the full collection metadata content here, just nested under `titles`. Add more metadata to the in the `query.response` part of the test to show you are filtering out everything but the title

You are looping through the collection metadata twice with two calls to `.map`, you don't need to have two of them

Reply:
Added more metadata to test, removed the titles and single loop in the inline function